### PR TITLE
LPS-149737 Add parameter when delete data provider to call DDMFormAdminPortlet

### PR DESCRIPTION
**Pull request by murilostodolni**
Related: https://issues.liferay.com/browse/LPS-149737

### CODE REVIEW

**Comment by murilostodolni** [comment]
ci:test:sf

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/resources/META-INF/resources/data_provider_action.jsp [hunk 1]
```diff
@@ -63,6 +63,7 @@ DDMDataProviderInstance ddmDataProviderInstance = (DDMDataProviderInstance)row.g
 		<portlet:actionURL name="/dynamic_data_mapping_data_provider/delete_data_provider" var="deleteURL">
 			<portlet:param name="redirect" value="<%= currentURL %>" />
 			<portlet:param name="dataProviderInstanceId" value="<%= String.valueOf(ddmDataProviderInstance.getDataProviderInstanceId()) %>" />
+			<portlet:param name="refererPortletName" value="com_liferay_dynamic_data_mapping_form_web_portlet_DDMFormAdminPortlet" />
```
[the remark this answers is not in the document]
Comment (murilostodolni) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/resources/META-INF/resources/data_provider_action.jsp:66]: Hi @carolmariaabb perfect, I didn't see this constant. I'm chaging now.

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**